src/client/io_tracker.cpp

```diff
@@ -505,7 +505,7 @@ void IOTracker::Done() {
     } else {
         MetricHelper::IncremUserEPSCount(fileMetric_, type_);
         if (type_ == OpType::READ || type_ == OpType::WRITE) {
-            LOG(ERROR) << "file [" << fileMetric_->filename << "]"
+            LOG(WARNING) << "file [" << fileMetric_->filename << "]"
```

Comment:
I think is not a good idea to lower this log level to WARNING, because when this error is printed, it means a real error happens and this error will return to the upper-layer application.